src/app/core/theme.component.ts

```diff
@@ -1,14 +1,16 @@
 import { NgIf } from '@angular/common';
 import { Component } from '@angular/core';
 import { SectionCardComponent } from 'src/app/features/section';
-
+import { ExampleDialogComponent } from '@shared/ui/modals/example dialog/example-dialog.component';
 @Component({
   selector: 'app-theme',
   standalone: true,
-  imports: [SectionCardComponent, NgIf],
+  imports: [SectionCardComponent, NgIf,ExampleDialogComponent],
   styles: ['.element { margin: 10px;}'],
   template: `
     <h1>Storybook-like route</h1>
+    <h3>Dialogi</h3>
+    <app-example-dialog></app-example-dialog>
```

Comment:
to tutaj w theme powinienes wywolac te 2 dialogi, oddzielnie input z dialogiemm, oddzielnie d ialogz buttonami; czyli logikę z example-dialog, przeniesc tutaj, a tamten folder wywalic